src/mesh/voxel_cloud.rs

```diff
@@ -685,6 +705,100 @@ impl ScalarField {
         }
         self.shrink_to_fit(volume_value_range_self)
     }
+    /// Computes interpolated values of two scalar fields. Prior to
+    /// interpolation, the scalar fields will be recomputed to distance fields.
+    /// The current scalar field will be mutated and resized to encompass values
+    /// from both scalar fields. The two scalar fields do not have to contain
+    /// voxels of the same size.  The interpolation is linear.
+    ///
+    /// `interpolation_factor` is a value between 0.0 and 1.0 defining the ratio
+    /// between the current value to the other value.
```

Comment:
Should we document what happens if the factor exceeds the 0..1 range?

Reply:
it will be a linear extrapolation, as expected. i believe there is no need to explicitly document that. i can, though